project/Build.scala

```diff
@@ -1397,6 +1405,23 @@ object Build {
         )
       }.value,
 
+      renderScaladocScalajsToFile := Def.inputTask {
+        val extraArgs = spaceDelimited("<arg>").parsed
+        val (destJS, destCSS, csses) = extraArgs match {
+          case js :: css :: tail => (js, css, tail)
+          case js :: Nil => (js, "", Nil)
+          case _ => throw new IllegalArgumentException("No js destination provided")
+        }
```

Comment:
It would be better to define a proper parser instead of using `spaceDelimited("<arg>")`. sbt parsers have nice autocompletion support.

Reply:
It's meant to be run from the bash script, I don't know if it's worth making a custom parser, though I can create one (I have no proficiency in custom parsers, so if you have a good tutorial, please share it with me)